### ISSUE TITLE
Dionaea not showing attacks

### ISSUE THREAD

**Issue by Norup123** [threatstream/mhn#710]
Hey. I have deployed a MHN server and another server with a couple of honeypots on it. Cowrie, conpot and dionaea. Attacks on the dionaea is not showing up for some reason. Cowrie and conpot is. I have dionaea expose port 80, and can connect to it from an outside network, so i know that its running
The mhn server:
![image](https://user-images.githubusercontent.com/54578187/64845074-8707dd00-d609-11e9-9c28-7c1be9b3f410.png)

This file is in ihandlers-enabled. The ipadress is correct, and the port is the same as the other 2 honeypots are using for the MHN server. 

I also tried connecting to the ftp that dionaea exposes, but that also doesnt show up on the MHN server.
![image](https://user-images.githubusercontent.com/54578187/64845180-c3d3d400-d609-11e9-99ee-7b07a2e766d9.png)

**Comment by neil-fox** [comment]
What cloud tech are you using to host your VM's?

**Comment by Norup123** [comment]
Running on my own servers, exposed with a 3g network

**Comment by neil-fox** [comment]
Ah right, I was thinking your issue may be firewall related if you were using something such as azure. The only connections i see for Dionaea on my attack map is for 445, do you have that port open?

**Comment by Norup123** [comment]
On the Dionaea? Or the MHN server? Everything nats to the honeypot machine. The Dionaea should feed on port 10000 right? That port is open on the mhn server. On the dionaea these are the services available
![image](https://user-images.githubusercontent.com/54578187/65030320-9e173980-d93f-11e9-94f1-68bfc74d447b.png)

So if i connect to port 80 i get the dionaea http pot, but nothing shows up on the mhn server

**Comment by neil-fox** [comment]
You're right Dionaea feeds back to the MHN server on 10000. However no connections to my Dionaea sensor on 80 show up on the attack map, only SMB/445 connections. Try a connection to Dionaea on 445 instead of 80 and see if that shows up on your map.

**Comment by Norup123** [comment]
How do i test that port 445? port 80 was easy with just the browser ^^

**Comment by neil-fox** [comment]
If you have a Kali instance try using Metasploit to test your Dionaea sensor:

https://www.adlice.com/catch-malware-honeypot/

Other than that I'm not sure what else to suggest i'm afraid.

**Comment by neil-fox** [comment]
Also whats the output of an nmap scan of your Dionaea sensor?

**Comment by Norup123** [comment]
Scanning the sensor machine shows all the open ports it should, and it shows smb1 on 445 etc.

Ill try testing port 445 with metasploit later.

So the issue is with dioneae only using some of the services for HPFeeds?

**Comment by neil-fox** [comment]
"So the issue is with dioneae only using some of the services for HPFeeds?" - Potentially, I'm in a similar position to yourself where I've only just started to play around with MHN and finding things out for myself through trial and error so I'm no expert on this. All i can say is that i only see 445 connections on my map for the Dionaea sensor so i know it definitely works.

**Comment by Norup123** [comment]
![image](https://user-images.githubusercontent.com/54578187/65037344-c0b04f00-d94d-11e9-912a-2f5f0d559487.png)
nmap scan shows this.
I dont really like that it says dionaea on port 443... Seems a bit stupid
I tried using metasploit to attack port 445, but i cant seem to get a connection through to it. The cowrie on the same machine works fine however.

Tested port 80 and that works aswell.
